cmd/format.go

```diff
@@ -52,12 +52,12 @@ func fixObjectSize(s int) int {
 	return s
 }
 
-func createStorage(fmt *meta.Format) (object.ObjectStorage, error) {
-	blob, err := object.CreateStorage(strings.ToLower(fmt.Storage), fmt.Bucket, fmt.AccessKey, fmt.SecretKey)
+func createStorage(fmat *meta.Format) (object.ObjectStorage, error) {
```

Comment:
Using `format` may be better in this case, because the word is just two letters longer.

Reply:
@Suave fix it.